test/api/v3/integration/user/auth/POST-register_local.test.js

```diff
@@ -416,5 +416,20 @@ describe('POST /user/auth/local/register', () => {
 
       expect(user.tags).to.not.be.empty;
     });
+
+    it('adds the correct tags to the correct tasks', async () => {
+      let user = await api.post('/user/auth/local/register', {
+        username,
+        email,
+        password,
+        confirmPassword: password,
+      });
+
+      let habits = await user.get('/tasks/user?type=habits');
```

Comment:
The above does not return a user attached to a request object. `user.get` is produced from our helper functions.